test/parallel/test-util-inspect.js

```diff
@@ -1253,18 +1257,36 @@ if (typeof Symbol !== 'undefined') {
 
 // Empty and circular before depth.
 {
-  const arr = [[[[]]]];
-  assert.strictEqual(util.inspect(arr), '[ [ [ [] ] ] ]');
-  arr[0][0][0][0] = [];
-  assert.strictEqual(util.inspect(arr), '[ [ [ [Array] ] ] ]');
-  arr[0][0][0] = {};
-  assert.strictEqual(util.inspect(arr), '[ [ [ {} ] ] ]');
-  arr[0][0][0] = { a: 2 };
-  assert.strictEqual(util.inspect(arr), '[ [ [ [Object] ] ] ]');
-  arr[0][0][0] = arr;
-  assert.strictEqual(util.inspect(arr), '<ref *1> [ [ [ [Circular *1] ] ] ]');
+  const arr = [[[[[[]]]]]];
+  assert.strictEqual(
+    util.inspect(arr),
+    '[\n  [\n    [ [ [ [] ] ] ]\n  ]\n]'
+  );
```

Comment:
Not particularily happy with how these are printed, by the way. It almost seems like the linebreak mechanism has some sort of bias towards depth of 2, but didn't investigate that yet.

Reply:
The default is currently set to `compact: 3`. That means the three most inner levels are combined on a single line in case they fit into `breakLength`. This has "five" levels (the most inner part counts as primitive, not as level). That's why the most outer two levels create the new lines.

Reply:
I see. Do you recall why the particular value of `3` was chosen? Was it to accomodate a depth of `2` or are they unrelated?

Reply:
Checking how many objects are all on the same line is difficult when it becomes to many. That's why I limited it to three by default. It is unrelated to the depth.